tests/test_rasterization.py

```diff
@@ -113,7 +113,78 @@ def test_rasterization_point_single_pixel():
 
     roi = GeoVector.from_bounds(xmin=0, ymin=0, xmax=5, ymax=5, crs=WEB_MERCATOR_CRS)
 
-    result = fc.rasterize(1, 1, bounds=roi).image
+    result = fc.rasterize(1, polygonize_width=1, bounds=roi).image
 
     assert_array_equal(result.data, expected_image.data)
     assert_array_equal(result.mask, expected_image.mask)
+
+
+@pytest.mark.parametrize("fill_value,dtype", [
+    (1, np.uint8),
+    (0, np.uint8),
+    (256, np.uint16),
+    (1.0, np.float32),
+    (1.5, np.float32),
+    (0.0, np.float32),
+    (256.0, np.float32)
+])
+def test_rasterization_function_user_dtype(fill_value, dtype):
+    resolution = 1
+
+    line = GeoVector.from_bounds(xmin=2, ymin=0, xmax=3, ymax=3, crs=DEFAULT_CRS)
+    roi = GeoVector.from_bounds(xmin=0, ymin=0, xmax=5, ymax=5, crs=DEFAULT_CRS)
+
+    fc = FeatureCollection.from_geovectors([line])
```

Comment:
why do you use a FeatureCollection here?  just to make a GeoFeature from the line GeoVector?

Reply:
you're right, I don't need to. the habit :) will change it